### PR TITLE
July 2020 refresh

### PR DESCRIPTION
**Pull request by paulopsgility**
Ready for SME review.

### CODE REVIEW

**Comment by paulopsgility** [comment]
@evmoreno Thank you for the feedback. Looking into it today.

**Comment by paulopsgility** [comment]
There are two separate labs in this workshop. When providing feedback please let us know which lab you are doing, the exercise name and number, AND the task name and number.

**Comment by paulopsgility** [comment]
@evmoreno we just updated the install. It should be working now.

**Comment by evmoreno** [comment]
Hi Paul apologise I am working through the Hands-on lab/Azure-Stack-Migrate/Before the HOL - Azure Stack Migrate.md. 

Prior to Task 4: Download and run the Azure Stack Hub Configurator Script

We need to run through the Post Deployment configuration - https://docs.microsoft.com/en-us/azure-stack/asdk/asdk-post-deploy?view=azs-2002 to register and install PowerShell on the ASDK Host machine.

Additional item to add

[System.Net.WebRequest]::DefaultWebProxy.Credentials = [System.Net.CredentialCache]::DefaultNetworkCredentials
Register-PSRepository -Default

Set-PSRepository -Name "PSGallery" -InstallationPolicy Trusted

As no PowerShell modules local.

This document now reviewed and input provided this document! I will move onto the next lab.

Cheers! Ev

**Comment by evmoreno** [comment]
Nearly - then we need to finish this by registering ASDK - https://docs.microsoft.com/en-us/azure-stack/asdk/asdk-register?view=azs-2002

**Comment by mwasham** [comment]
The Azure Stack configurator script installs the PowerShell Modules for you when you run it. It also registers Azure Stack with Azure AD so both of those steps are covered in Task 4.

File: Hands-on lab/Azure-Stack-Migrate/Before the HOL - Azure Stack Migrate.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
+﻿![Microsoft Cloud Workshops](https://github.com/Microsoft/MCW-Template-Cloud-Workshop/raw/master/Media/ms-cloud-workshop.png)
+ 
+<div class="MCWHeader1">
+Azure Stack Migrate
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/Azure-Stack-Migrate/Before the HOL - Azure Stack Migrate.md:4]: Azure stack migrate or operate (below says operate)?

File: Hands-on lab/Azure-Stack-Migrate/HOL step-by-step - Azure Stack Migrate.md [hunk 2]
```diff
@@ -0,0 +1,681 @@
...
+
+12. Locate **Connection strings** section below **Application settings** in the Azure Stack Hub User portal, select **+ New connection string** and add a new connection string with the following values then select **OK**. 
+
+   - Name: **ContosoFinance (must match exactly -- case sensitive)**
+
+   - Value: **Enter the Connection String for the SQL Database in Azure Stack Hub you created earlier in this lab**.
+
+   - Type: **SQLAzure**
+
+    **Note:** You may receive an error message that says *value is a required property on all connection string*. This can be ignored.
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/Azure-Stack-Migrate/HOL step-by-step - Azure Stack Migrate.md:577]: Check formatting please.  I can see the asterisks when I view the file.

File: Hands-on lab/Azure-Stack-Migrate/HOL step-by-step - Azure Stack Migrate.md [hunk 3]
```diff
@@ -0,0 +1,681 @@
...
+
+3. Add a new **Application Setting** with the following values and select **OK**. 
+
+    - App Setting Name: **offersAPIUrl**
+
+    - Value: Enter the **HTTPS** URL for the Offers API App with **/api/get** appended to the end.
+    
+    Example: <https://contosofinanceapi.appservice.local.azurestack.external/api/get>
+
+       ![Under App settings, offersAPIUrl and its URL are selected.](images/Hands-onlabstep-by-step-AzureStackimages/media/image136.png "App settings")
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/Azure-Stack-Migrate/HOL step-by-step - Azure Stack Migrate.md:467]: Check formatting please, Image not resolving in view.

File: Hands-on lab/Azure-Stack-Migrate/HOL step-by-step - Azure Stack Migrate.md [hunk 4]
```diff
@@ -0,0 +1,681 @@
...
+
+    ![The URL is selected in the App Service blade.](images/Hands-onlabstep-by-step-AzureStackimages/media/image127b.png "App Service blade")
+
+### Task 3: Update the Application Settings of the Web App with the API URL
+
+1. Open the ContosoFinanceWeb application in the Azure Stack Hub User portal and select **Configuration** on the left under **Settings**.
+
+    ![In the App Service blade, under settings, Application settings is selected.](images/Hands-onlabstep-by-step-AzureStackimages/media/image134.png "App Service blade")
+
+2. Scroll down and locate the **Application settings** section
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/Azure-Stack-Migrate/HOL step-by-step - Azure Stack Migrate.md:455]: Missing period

File: Hands-on lab/Azure-Stack-Migrate/HOL step-by-step - Azure Stack Migrate.md [hunk 5]
```diff
@@ -0,0 +1,681 @@
...
+## Requirements
+
+-   A Microsoft Azure subscription.
+
+## Exercise 1: Create Azure Stack Hub Deployment Taxonomy for Tenants
+
+In this exercise, you will create the subscriptions, offers, and plans that will be used by your Azure Stack Hub tenants. 
+
+Duration: 15-30 minutes
+
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/Azure-Stack-Migrate/HOL step-by-step - Azure Stack Migrate.md:97]: Need a  Task 1: Title please

File: Hands-on lab/Azure-Stack-Migrate/HOL step-by-step - Azure Stack Migrate.md [hunk 6]
```diff
@@ -0,0 +1,681 @@
...
+<!-- TOC -->
+
+- [Azure Stack Migrate hands-on lab step-by-step](#azure-stack-migrate-hands-on-lab-step-by-step)
+  - [Abstract and learning objectives](#abstract-and-learning-objectives)
+  - [Overview](#overview)
+  - [Solution architecture](#solution-architecture)
+  - [Help references](#help-references)
+  - [Requirements](#requirements)
+  - [Exercise 1: Create Azure Stack Hub Deployment Taxonomy for Tenants](#exercise-1-create-azure-stack-hub-deployment-taxonomy-for-tenants)
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/Azure-Stack-Migrate/HOL step-by-step - Azure Stack Migrate.md:35]: Need a Task 1: Title

File: Hands-on lab/Azure-Stack-Operate/Before the HOL - Azure Stack Operate.md [hunk 7]
```diff
@@ -0,0 +1,206 @@
...
+
+© 2020 Microsoft Corporation. All rights reserved.
+
+Microsoft and the trademarks listed at <https://www.microsoft.com/en-us/legal/intellectualproperty/Trademarks/Usage/General.aspx> are trademarks of the Microsoft group of companies. All other trademarks are property of their respective owners.
+
+**Contents**
+
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/Azure-Stack-Operate/Before the HOL - Azure Stack Operate.md:27]: Need TOC

File: Hands-on lab/Azure-Stack-Operate/Before the HOL - Azure Stack Operate.md [hunk 8]
```diff
@@ -0,0 +1,206 @@
...
+### Task 1: Provision an Azure VM to Host the Azure Stack Hub Development Kit
+
+In this task, you will provision an Azure VM that will host the Azure Stack Development Kit deployment.
+
+Execute the following PowerShell code. When prompted for Azure credentials, you will need to authenticate with an Azure Account that has Global Administrator access on the Azure AD Tenant as well as Owner rights on the Azure subscription. 
+
+There will be a second prompt for the local administrator account. Specify demo@pass123 when prompted. 
+
+> Note: Ensure the Azure Subscription has enough cores available on the quota to deploy a VM sized: Standard E20s v3.
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/Azure-Stack-Operate/Before the HOL - Azure Stack Operate.md:49]: >**Note**: 
Missing bold format

File: Hands-on lab/Azure-Stack-Operate/HOL step-by-step - Azure Stack Operate.md [hunk 9]
```diff
@@ -0,0 +1,1471 @@
...
+
+3. On the Offers blade, select **+ Add**.
+
+4. On the Create a new offer blade, select **Delegated offer**.
+
+5. On the Delegated offer blade, select the entry corresponding to the delegated offer based on **services-for-dp-offer1**.
+
+6. On the Create a new offer blade, specify the following settings:
+
+    - Display name: the name matching the **Delegated offer** entry.
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/Azure-Stack-Operate/HOL step-by-step - Azure Stack Operate.md:895]: The rest of the document capitalizes the first word after the colon when it's a complete sentence.  Please do the same for these and check the rest of the document for any others that slipped in.

File: README.md [hunk 10]
```diff
@@ -2,7 +2,7 @@
 
 Contoso Finance is one of the largest banks in the United States with a significant amount of their revenue coming from their residential mortgage business. As part of Contoso's shift to a cloud first strategy they planning to migrate their loan web applications to a hybrid cloud solution. During the planning stages, Contoso realized they would not be able to retain their customer data in US based Azure regions due to corporate compliance policies and regulatory issues. They have selected Azure Stack as the deployment method to take advantage of Azure technologies while still maintaining compliance.
 
-January 2020
+July 2020
 
 ## Target audience
```
Comment (DawnmarieDesJardins) [inline README.md:8]: Random 3am thought, sorry.  If the two labs have different abstracts, this page will need to be updated to show both.  Thanks!

File: Hands-on lab/Azure-Stack-Migrate/Before the HOL - Azure Stack Migrate.md (hunk 1 above)
Comment (paulopsgility) [reply Hands-on lab/Azure-Stack-Migrate/Before the HOL - Azure Stack Migrate.md:4]: Fixed

File: Hands-on lab/Azure-Stack-Migrate/HOL step-by-step - Azure Stack Migrate.md (hunk 2 above)
Comment (paulopsgility) [reply Hands-on lab/Azure-Stack-Migrate/HOL step-by-step - Azure Stack Migrate.md:577]: Fixed.

File: Hands-on lab/Azure-Stack-Migrate/HOL step-by-step - Azure Stack Migrate.md (hunk 3 above)
Comment (paulopsgility) [reply Hands-on lab/Azure-Stack-Migrate/HOL step-by-step - Azure Stack Migrate.md:467]: Fixed.

File: Hands-on lab/Azure-Stack-Migrate/HOL step-by-step - Azure Stack Migrate.md (hunk 5 above)
Comment (paulopsgility) [reply Hands-on lab/Azure-Stack-Migrate/HOL step-by-step - Azure Stack Migrate.md:97]: Fixed.

File: Hands-on lab/Azure-Stack-Migrate/HOL step-by-step - Azure Stack Migrate.md (hunk 4 above)
Comment (paulopsgility) [reply Hands-on lab/Azure-Stack-Migrate/HOL step-by-step - Azure Stack Migrate.md:455]: Fixed.

File: Hands-on lab/Azure-Stack-Migrate/HOL step-by-step - Azure Stack Migrate.md (hunk 6 above)
Comment (paulopsgility) [reply Hands-on lab/Azure-Stack-Migrate/HOL step-by-step - Azure Stack Migrate.md:35]: Fixed.

File: Hands-on lab/Azure-Stack-Operate/Before the HOL - Azure Stack Operate.md (hunk 7 above)
Comment (paulopsgility) [reply Hands-on lab/Azure-Stack-Operate/Before the HOL - Azure Stack Operate.md:27]: Fixed.

File: Hands-on lab/Azure-Stack-Operate/HOL step-by-step - Azure Stack Operate.md (hunk 9 above)
Comment (paulopsgility) [reply Hands-on lab/Azure-Stack-Operate/HOL step-by-step - Azure Stack Operate.md:895]: Fixed

**Pull request closed without merging**